games/devtest/mods/testentities/models/testentities_lava_flan.x

```diff
@@ -0,0 +1,3506 @@
+xof 0303txt 0032
```

Comment:
can't we convert this to binary? it's quite big...

Reply:
I should have used `.gitattributes` properly here.
I think it is a GitHub limitation that this is not respected.
Locally it is hidden in the diff as a "binary file" just fine for me.

I think in terms of file size, this is acceptable for a model file - it is around the size of Sam.

I would also want to keep this as a text file (and ideally as the exact file used in mobs_redo) since these take slightly different code paths. (And I don't have software set up to reliably convert it into a binary .x file).